Entity/BaseProduct.php

```diff
@@ -51,7 +51,7 @@
     /**
      * @var double
```

Comment:
@michaelzangerle please change it here! we should stay with the phpdoc standard. also we should be avoid that this will not be regenerated sometime!